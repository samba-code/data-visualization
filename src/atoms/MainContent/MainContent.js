import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

const StyledMain = styled.div`
  width: 90%;
  max-width: 1024px;
`;

const MainContent = ({ children }) => <StyledMain>{children}</StyledMain>;

MainContent.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ]).isRequired,
};

export default MainContent;
