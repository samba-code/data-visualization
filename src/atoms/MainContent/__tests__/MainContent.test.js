import React from "react";
import { render } from "enzyme";

import MainContent from "../MainContent";

describe("MainContent", () => {
  it("should match snapshot", () => {
    const wrapper = render(<MainContent />);
    expect(wrapper).toMatchSnapshot();
  });
});
