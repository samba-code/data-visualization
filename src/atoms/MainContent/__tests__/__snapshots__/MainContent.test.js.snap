// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent should match snapshot 1`] = `
<div
  class="MainContent__StyledMain-sc-1ihw2m3-0 gLWapz"
>
  Some Content
</div>
`;
